### PR TITLE
helm: add TTL configuration on value.yaml and update helm docs

### PR DESCRIPTION
**Pull request by cwen0**
Signed-off-by: cwen0 <cwenyin0@gmail.com>

### What problem does this PR solve?

 

Add TTL configuration on `value.yaml` and update helm docs

### Related changes

- [ ] Need to update `chaos-mesh/website`
- [ ] Need to update `Dashboard UI`
- Need to **cheery-pick to release branches**
  - [ ] release-2.1
  - [ ] release-2.0

### Checklist

Tests

- [ ] Unit test
- [ ] E2E test
- [ ] No code
- [ ] Manual test (add steps below)

Side effects

- [ ] Breaking backward compatibility

### Release note 

```text
Please add a release note.

You can safely ignore this section if you don't think this PR needs a release note.
```

### DCO

If you find the DCO check fails, please run commands like below (Depends on the actual situations. For example, if the failed commit isn't the most recent) to fix it:

```shell
git commit --amend --signoff
git push --force
```

### CODE REVIEW

File: helm/chaos-mesh/README.md [hunk 1]
```diff
@@ -87,6 +87,9 @@ The following tables list the configurable parameters of the Chaos Mesh chart an
 | `dashboard.env.LISTEN_PORT` | | `2333` |
 | `dashboard.env.DATABASE_DRIVER`| The db drive used for Chaos Dashboard, support db: sqlite3, mysql| `sqlite3` |
 | `dashboard.env.DATABASE_DATASOURCE`| The db dsn used for Chaos Dashboard | `/data/core.sqlite` |
+| `dashboard.env.CLEAN_SYNC_PERIOD`| Set the sync period to clean up archived data | `12h` |
+| `dashboard.env.TTL_EVENT`| Set TTL of archived event data | `168h` |
+| `dashboard.env.TTL_EXPERIMENT`| Set TTL of archived experiment data | `336` |
```
Comment (fewdan) [inline helm/chaos-mesh/README.md:92]: Should it be '336h' instead of '336'?

File: helm/chaos-mesh/values.yaml [hunk 2]
```diff
@@ -295,6 +295,13 @@ dashboard:
     # you set a database encryption secret. This must be set before any secrets are stored
     # in the database.
     # DATABASE_SECRET:
+
+    # Set the sync period to clean up archived data
+    CLEAN_SYNC_PERIOD: 12h
+    # Set TTL of archived event data
+    TTL_EVENT: 168h
```
Comment (iguoyr) [inline helm/chaos-mesh/values.yaml:302]: Could we use `7d` and `14d` instead? `168h` is not intuitive 🤔

File: helm/chaos-mesh/README.md (hunk 1 above)
Comment (cwen0) [reply helm/chaos-mesh/README.md]: fixed

File: helm/chaos-mesh/values.yaml (hunk 2 above)
Comment (cwen0) [reply helm/chaos-mesh/values.yaml:302]: Yes, I will improve it in another single pr.

**Comment by iguoyr** [comment]
@cwen0 `install.sh` need to be updated.

**Comment by cwen0** [comment]
> @cwen0 `install.sh` need to be updated.

Fixed! PTAL again, Thanks!

**Review by iguoyr — APPROVED** [review]
LGTM

**Comment by cwen0** [comment]
@fewdan PTAL again! Thanks!

**Review by fewdan — APPROVED** [review]
LGTM

**Pull request merged**